website_field_autocomplete/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Website Field - AutoComplete",
+    "summary": 'Provides an autocomplete field for Website on any model',
+    "version": "11.0.1.0.0",
+    "category": "Website",
+    "website": "https://laslabs.com/",
```

Comment:
Change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)